### ISSUE TITLE
Make "Always show Markdown symbols" option false by default

### ISSUE THREAD

**Issue by orlowdev** [ordo-pink/ordo-electron#30]
If the option is enabled, md symbols are always shown (what a surprise!). Should be off by default because the only reason for them to be shown is a11y which is not the most common necessity. Can be assigned in `UserSettings` as a "0.1.1" migration.
